pkg/lib/env.go

```diff
@@ -27,6 +27,7 @@ const (
 	WindowsSysprepFSS             = "FSS_WCP_WINDOWS_SYSPREP"
 	VMServiceBackupRestoreFSS     = "FSS_WCP_VMSERVICE_BACKUPRESTORE"
 	MaxCreateVMsOnProviderEnv     = "MAX_CREATE_VMS_ON_PROVIDER"
+	WCPServiceUserEnv             = "WCPSVC_USER"
```

Comment:
Please document either this constant or the function `GetWCPServiceSolutionUser` explaining what the value of the env var is and how it is used.